plugins/clean.py

```diff
@@ -17,31 +17,47 @@ def handle(self, directive, data):
 
     def _process_clean(self, targets):
         success = True
-        for target in targets:
-            success &= self._clean(target)
+        defaults = self._context.defaults().get(self._directive, {})
+        force = defaults.get('force', False)
+        for target in self._dispatch_targets(targets):
```

Comment:
Came up with this solution after writing my own ```aptget``` plugin. I agree, as far as we don't modify the dictionary and accept only lists or dictionaries, ```_dispatch_targets``` method is unnecessary here.